lib/main_common.pm

```diff
@@ -998,6 +998,10 @@ sub load_inst_tests {
         loadtest "installation/secure_boot";
     }
     if (installyaststep_is_applicable()) {
+        # Do not repeat step when change_desktop has been executed
+        if (check_var("ENABLE_DEPENDENCY_RESOLVER", "1")) {
```

Comment:
Do you already have a list of test suites which need this variable to be set? I guess we can set them in advance and then merge.